src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/PhysicalMarker/QRCodeMarkerDetectorSpatialLocalizer.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.SpectatorView
+{
+    public class QRCodeMarkerDetectorSpatialLocalizer : MarkerDetectorSpatialLocalizer

```

Comment:
>QRCodeMarkerDetectorSpatialLocalizer [](start = 17, length = 36)

Do QR codes use the same [0-n] index-based ID system? Thinking about whether or not different types of marker detectors might warrant different settings.

Reply:
yeah they still use that as well

Reply:
qr codes used are a "sv" prefix followed by an index. Right now we only support up to index 19.